nautobot/extras/templates/extras/job_detail.html

```diff
@@ -284,6 +284,32 @@
                     {% endif %}
                 </td>
             </tr>
+            <tr>
+                <td>Default Job Queue</td>
+                <td>
+                    {% if object.default_job_queue %}
+                        {{ object.default_job_queue|hyperlinked_object }}
+                    {% else %}
+                        {{ None|placeholder }}
+                    {% endif %}
+                </td>
+                <td>
+                    {% if object.default_job_queue_override %}
+                        <span class="text-muted">
+                            overridden; default is:&nbsp;
```

Comment:
The non-breaking space (`&nbsp;`) here isn't doing much since it's immediately followed by other whitespace.